docs/source/API/core/builtinreducers/Sum.rst

```diff
@@ -16,84 +13,94 @@ Usage
 
 .. code-block:: cpp
 
-    T result;
-    parallel_reduce(N,Functor,Sum<T,S>(result));
+   T result;
+   parallel_reduce(N,Functor,Sum<T,S>(result));
```

Comment:
Is the 3 spaces indentation intentional?

Reply:
i asked Antoine to do that to aling with the "c" of the code-block but we can change if needed

Reply:
Yes it's intentional, I stay tuned if you want to change

Reply:
I am not a big fan of it but I won't block because I don't know what is the common practice with restructured text and I don't want to add inconsistencies to the doc source.